src/common/regions/index.ts

```diff
@@ -29,8 +29,10 @@ export abstract class Region {
   abstract get abbreviation(): string;
   abstract get relativeUrl(): string;
 
+  // Todo (Chelsi) - switch back before merging (or maybe keep this change)
   get canonicalUrl() {
-    return urlJoin('https://covidactnow.org', this.relativeUrl);
+    // return urlJoin('https://covidactnow.org', this.relativeUrl);
+    return urlJoin(window.location.origin, this.relativeUrl);
```

Comment:
I _think_ we can keep this change (we will need to test it with embeds)

Reply:
reverting for now